drv/lpc55-swd/build.rs

```diff
@@ -88,6 +103,161 @@ fn generate_swd_functions(config: &TaskConfig) -> Result<()> {
     Ok(())
 }
 
+fn prepare_endoscope() -> Result<(), anyhow::Error> {
+    let key = "CARGO_BIN_FILE_ENDOSCOPE";
+    let elf_path = PathBuf::from(
+        std::env::var(key)
+            .with_context(|| format!("Cannot read env var '{}'", key))?,
+    );
```

Comment:
To make sure that this is rebuilt if the environmental variables changes, you should add
```rust
    println!("cargo:rerun-if-env-changed={key}");
```

Reply:
Done. Thanks!